app/models/edm/web_resource.rb

```diff
@@ -11,8 +11,7 @@ class WebResource
     mount_uploader :media, MediaUploader
 
     belongs_to :edm_rights,
-               class_name: 'CC::License', inverse_of: :edm_rights_for_edm_web_resources,
-               optional: true
+               class_name: 'CC::License', inverse_of: :edm_rights_for_edm_web_resources
```

Comment:
`edm:rights` is an *optional* property on `edm:WebResource`. This should be reflected in the model, and requirement enforced elsewhere, e.g. in per-campaign validation rules.